tests/DCPS/GroupPresentation/run_test.pl

```diff
@@ -37,54 +37,15 @@
 unlink $dcpsrepo_ior;
 unlink <*.log>;
 
-$DCPSREPO = PerlDDS::create_process ("$ENV{DDS_ROOT}/bin/DCPSInfoRepo",
-                                     "-ORBDebugLevel 10 -ORBVerboseLogging 1 -DCPSDebugLevel $debuglevel -ORBLogFile DCPSInfoRepo.log -o $dcpsrepo_ior -NOBITS");
+my $test = new PerlDDS::TestFramework();
```

Comment:
This is one of the test scripts that hasn't yet been migrated over to TestFramework.  I'm fine with doing it now but I suggest adding `use strict;` to catch any potential problems.

Reply:
Sure. I only modified this because TSAN was reporting some odd lock inversion issues (I think in the InfoRepo) which seemed related to how it was being shut down... I wondered if using the tools in setup_discovery would resolve it, which it did.